src/runtime_src/core/tools/xbmgmt2/flash/flasher.cpp

```diff
@@ -97,7 +97,7 @@ Flasher::E_FlasherType Flasher::getFlashType(std::string typeStr)
  * upgradeFirmware
  */
 int Flasher::upgradeFirmware(E_FlasherType flash_type,
-    firmwareImage *primary, firmwareImage *secondary)
+    firmwareImage *primary, firmwareImage *secondary, firmwareImage* stripped)
```

Comment:
We have run out of time for this release so what I am about to mentioned cannot be done for this release....
This code should have been updated to pass references or use shared pointers.